testgrid/specs/storage-migration.yaml

```diff
@@ -831,3 +825,53 @@
       echo Rook namespace found after the upgrade.
       exit 1
     fi
+- name: Migrate from OpenEBS (S3 disabled) to OpenEBS + Minio
+  flags: yes
+  installerSpec:
+    kubernetes:
+      version: 1.26.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.1
+    kotsadm:
+      version: latest
+      disableS3: true
+    ekco:
+      version: latest
+    openebs:
+      version: 3.4.x
+      isLocalPVEnabled: true
+      localPVStorageClassName: local
+  upgradeSpec:
+    kubernetes:
+      version: 1.26.x
+    containerd:
+      version: 1.6.x
+    flannel:
+      version: 0.21.x
+    registry:
+      version: 2.8.1
+    kotsadm:
+      version: latest
+    ekco:
+      version: latest
+    openebs:
+      version: 3.4.x
+      isLocalPVEnabled: true
+      localPVStorageClassName: local
+    minio:
+      version: latest
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    create_deployment_with_mounted_volume "migration-test" "default" "/data" "registry:2.8.1"
+    create_random_file_and_upload_to_deployment "migration-test" "default" "./test.data" "/data/test.data"
+    test_push_image_to_registry
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    sleep 120
+    download_file_from_deployment_and_compare "migration-test" "default" "./test.data" "/data/test.data"
+    pvc_uses_provisioner "migration-test" "default" "openebs"
+    test_pull_image_from_registry
```

Comment:
Can we add here the check to ensure that the rook-ceph ns was removed?
Only that then it shows fine.

Reply:
Rook is not installed on this test so I don't think we actually need that.